core/payment/src/api/debit_notes.rs

```diff
@@ -331,14 +359,24 @@ async fn accept_debit_note(
         }
         .await
         {
-            Ok(_) => response::ok(Null),
+            Ok(_) => {
+                log::info!("DebitNote [{}] accepted.", path.debit_note_id);
+                response::ok(Null)
+            }
```

Comment:
ditto: move counter here & introduce counting errors as they are divided into `40x` and `50x`